README.md

```diff
@@ -53,7 +53,12 @@ If you run into problems using the SDK, you can
 * Post [issues][github-issues] here on GitHub (for code errors)
  
 ## Getting started
- 
+
+### Prerequisites
+
+* JDK 8 or later
+* Apache Maven (3.6.x or later)
```

Comment:
Any build system could be used, gradle, ivy, etc. (though we likely would only support Maven or Gradle)

Reply:
so we'll add a "recommended" next to Maven?

Reply:
I'd just leave the build system off completely, or maybe just say the artifacts are published to [Maven Central](https://search.maven.org/search?q=g:com.okta.sdk)

(it's similar to the Node JS world, you could use yarn or NPM, it's _mostly_ an implementation detail)

Reply:
done